CHANGELOG.md

```diff
@@ -187,6 +187,9 @@ v1.0.0-dev (unreleased)
 -   Introduced OpenTracing helpers for transport authors.
 -   Created the `yarpc.Serialize` package for marshalling RPC messages at rest.
     Useful for transports that persist RPC messages.
+-   Tranports have access to `DispatchOnewayHandler` and `DispatchUnaryHandler`.
+    These should be called by all `transport.Inbounds` instead of directly
+    calling handlers.
```

Comment:
maybe even add something along the line of: These functions shield YARPC from handler panics and other common error conversion.